evap/results/exporters.py

```diff
@@ -122,53 +133,56 @@ def filter_evaluations(semesters, evaluation_states, degrees, course_types, cont
                 continue
             if not evaluation.can_publish_rating_results and not include_not_enough_voters:
                 continue
-            results = OrderedDict()
+            results: OrderedDict[int, list[QuestionResult]] = OrderedDict()
             for contribution_result in get_results(evaluation).contribution_results:
                 for questionnaire_result in contribution_result.questionnaire_results:
                     # RatingQuestion.counts is a tuple of integers or None, if this tuple is all zero, we want to exclude it
+                    question_results: list[QuestionResult] = questionnaire_result.question_results
                     if all(
-                        not question_result.question.is_rating_question
-                        or question_result.counts is None
-                        or sum(question_result.counts) == 0
-                        for question_result in questionnaire_result.question_results
+                        not isinstance(question_result, RatingResult)
+                        or (question_result.counts is None or sum(question_result.counts) == 0)
+                        for question_result in question_results
                     ):
                         continue
                     if (
                         not contributor
                         or contribution_result.contributor is None
                         or contribution_result.contributor == contributor
                     ):
-                        results.setdefault(questionnaire_result.questionnaire.id, []).extend(
-                            questionnaire_result.question_results
-                        )
+                        results.setdefault(questionnaire_result.questionnaire.id, []).extend(question_results)
                         used_questionnaires.add(questionnaire_result.questionnaire)
-            evaluation.course_evaluations_count = evaluation.course.evaluations.count()
-            if evaluation.course_evaluations_count > 1:
+            monkeypatched_evaluation: Any = evaluation
```

Comment:
["Monkeypatched"](https://en.wikipedia.org/wiki/Monkey_patch) probably isn't the most precise word for what we're doing here. What do you think about `evaluation_as_any`?